worlds/doom_1993/Rules.py

```diff
@@ -0,0 +1,139 @@
+# This file is auto generated. More info: https://github.com/Daivuk/apdoom
+
+from typing import TYPE_CHECKING
+from worlds.generic.Rules import set_rule
+
+if TYPE_CHECKING:
+    from . import DOOM1993World
+
+
+def set_rules(doom_1993_world: "DOOM1993World"):
+    player = doom_1993_world.player
+    world = doom_1993_world.multiworld
+
+    # Hangar (E1M1) - E1M1
```

Comment:
This method follows a very repetitive pattern of
```
# comment about what the following rules are for
rules
```
I would suggest extracting each individual block to a method, named after what it does. So the first one would be
`set_e1m1_hangar_rules()` or `set_hangar_e1m1_rules()`

This would make the file much easier to manage, while removing the need for each of these comments, as the naming would make it very clear

Reply:
This part is generated code. I won't touch for now, but keep note for the future as I am refactoring the tool that generates the scripts.